SPEC.md

```diff
@@ -334,32 +379,79 @@ interpreted as described in the following table:
 | Carriage Return               | `\r`   | `U+000D` |
 | Character Tabulation (Tab)    | `\t`   | `U+0009` |
 | Reverse Solidus (Backslash)   | `\\`   | `U+005C` |
-| Solidus (Forwardslash)        | `\/`   | `U+002F` |
 | Quotation Mark (Double Quote) | `\"`   | `U+0022` |
 | Backspace                     | `\b`   | `U+0008` |
 | Form Feed                     | `\f`   | `U+000C` |
-| Unicode Escape                | `\u{(1-6 hex chars)}` | Code point described by hex characters, up to `10FFFF` |
+| Unicode Escape                | `\u{(1-6 hex chars)}` | Code point described by hex characters, as long as it represents a [Unicode Scalar Value](https://unicode.org/glossary/#unicode_scalar_value) |
+| Whitespace Escape             | See below | N/A   |
```

Comment:
To help with marking a dedent, do we want to add an escape code for a normal space? I suggest just `\s`. That way you can escape the first significant space easily, without having to remember that it's U+0020 (and `\s` is easier to write than `\u{20}`).

Reply:
Oh crap, I didn't even realize it wasn't present in the table either, I added ALL the usual escapes (\s\r\n\f\b)

https://github.com/IceDragon200/kuddle/pull/1/files#diff-1663e36dafaff337ff4883c20f375c3cebf1de34e44f592012ddb870e8e29651R196-R214

Maybe we should add it, it doesn't really hurt anything

Reply:
I like this idea. It may be a way for us to work around the issue @IceDragon200 brought up about dedenting text that should have _some_ global indentation (the suggested alternative being to determine dedent level based on the closing `"`'s indentation, like Elixir does).

Reply:
this is done now